surveying_dust_structure_via_GALEX_MIS_pt1/surveying_dust_structure_via_GALEX_MIS_pt1.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>What is triggered star formation? It sounds interesting; you should add a link to more information</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>